library/LuaApi.cpp

```diff
@@ -3577,6 +3577,67 @@ static int internal_md5file(lua_State *L)
     }
 }
 
+/**************************
+ * Wrappers for Alias API *
+ **************************/
+
+static int internal_getAliasCommand(lua_State *L)
+{
+    const char *name = luaL_checkstring(L, 1);
+    std::string ret = Core::getInstance().GetAliasCommand(name);
+    if (!ret.empty())
+        lua_pushstring(L, ret.c_str());
+    else
+        lua_pushnil(L);
+    return 1;
+}
+
+static int internal_listAliases(lua_State *L)
+{
+    auto aliases = Core::getInstance().ListAliases();
```

Comment:
If we add a `Lua::Push` override for `std::vector`, this function could be reduced to `Lua::Push(L, Core::getInstance().ListAliases()); return 1;`

Reply:
Nice - will look at this.